task_schedule.cfg

```diff
@@ -39,17 +39,20 @@ alert       aca@head.cfa.harvard.edu
 # executed only once for each <number> of times the task is executed.  In the
 # example below, the commands are done once each 1, 2, and 4 minutes, respectively.
 
+# The commented-out scripts are run in Ska2 with mica version 3.17.  The uncommented
+# one(s) are run with this task_schedule.cfg in Ska3 with mica version 4.x.
+
 <task mica_archive>
       cron       * * * * *
       check_cron * * * * *
-      exec 1: update_obspar.py
-      exec 1: update_aca_l0.py
-      exec 1: update_asp_l1.py
+      # exec 1: update_obspar.py
+      # exec 1: update_aca_l0.py
+      # exec 1: update_asp_l1.py
       exec 1: update_starchecks.py
```

Comment:
So my plan was to put this in its own task schedule file ... would you prefer I do it that way or set up multiple tasks in a single task schedule (we haven't really done that in a while)?